script/countries.py

```diff
@@ -0,0 +1,27 @@
+"""Helper script to update country list.
+
+ISO does not publish a machine readable list free of charge, so the list is generated
+with help of the pycountry package.
+"""
+import pathlib
+
+import black
+import pycountry
+
+BASE = """
+\"\"\"Automatically generated by currencies.py.
+
+The values are directly corresponding to the ISO 3166 standard. If you need changes
+to the political situation in the world, please contact the ISO 3166 working group.
+
+To update, run python3 -m script.currencies
+\"\"\"
```

Comment:
References to `currencies` need adjusting

Reply:
This has been fixed